### ISSUE TITLE
Add more settings

### ISSUE THREAD

**Issue by nightsha-de** [nightsha-de/npm-vwconnectapi#9]
Add the ability to set autoUnlockPlugWhenCharged, maxChargeCurrentAC, climatisationWithoutExternalPower, climatizatioAtUnlock and windowHeatingEnabled options (as available in the WeConnectId app).

**Comment by pixxelfriend** [comment]
I guess it would be nicer to pass the options via constructor directly, like the developers from the io broker plugin did. Its easier to use and it would reduce the amount code a bit, because the setter methods will not be needed anymore.  If you like, i can create another pull request.

**Comment by nightsha-de** [comment]
I agree for the basic mandatory parameters (e.g. username, password, car type). For everything else we'll need the setter methods anyway or you won't be able to set a different target SOC or target temperature for example without instantiating a new object of the class. But feel free to create a pull request with your proposal and we discuss it there.

This issue was intended as a reminder for me to add the ability to set autoUnlockPlugWhenCharged, maxChargeCurrentAC, climatisationWithoutExternalPower, climatizatioAtUnlock and windowHeatingEnabled options (as available in the WeConnectId app). Unfortunately I'm a bit occupied with work and offline projects lately so bear with me if responses take a little longer sometimes.
